roles/cfg_openwrt/tasks/main.yml

```diff
@@ -12,6 +12,7 @@
     apply:
       tags: always
   tags: always
+  when: os == "openwrt" | default ("openwrt")
```

Comment:
I think there are much better ways to archieve that. Lets have a workshop or chat on that.